src/ui/atoms/listItem/listItem.scss

```diff
@@ -1,40 +1,79 @@
 @import '../../styles/themes.scss';
+@import '../../styles/animations.scss';
 
 .okp4-listitem-main {
-  display: grid;
-  column-gap: 10px;
-  padding: 10px;
-  align-items: center;
-  grid-template-columns: minmax(30px, 8%) auto auto;
-  grid-template-rows: repeat(2, auto);
-  grid-template-areas:
-    'leftElement title rightElement'
-    'leftElement description rightElement';
-
   @include with-theme {
     background-color: themed(inverted-primary-background);
+    display: grid;
     border-radius: 10px;
-  }
+    gap: 10px;
+    padding: 10px;
+    overflow: auto;
 
-  .okp4-listitem-title {
-    grid-area: title;
-    justify-content: start;
-    margin: 3px;
-  }
+    &.clickable:hover {
+      cursor: pointer;
+      box-shadow: 0 2px 4px themed(inverted-field);
+      animation: 0.2s ease-out hover-animation;
```

Comment:
Here you can do `animation: 0.2s ease-out hover-animation forwards;` . With the "forwards" at the end of the ligne, the state will persist once the animation is done. If you implement it, you can delete the line just below.

https://developer.mozilla.org/en-US/docs/Web/CSS/animation-fill-mode

Reply:
Cool I didn't know about this property, thanks! 😃